core/test/routing.spec.js

```diff
@@ -10,14 +10,17 @@ import { afterEach } from 'mocha';
 
 describe('Routing', () => {
   let component;
+  let lastObj;
```

Comment:
I would rewrite this part as follows:
```
beforeEach(() => {
  let lastObj = {};
  component = {
    set: obj => {
      Object.assign(lastObj, obj);
    },
    get: () => lastObj
  };
});
```
Like that:
* Defining the `lastObj` variable makes clear you are not using it anywhere outside the `beforeEach` block in the spec file.
* You can define the get function once instead of redefining it every time the set function is called 
* No need to reassign `lastObj`, since Object.assign sets the merged object to its first parameter.